CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # v1.16
 
+## Release v1.16.7 - 2025/01/29
+
+### Bug Fix
+
+* windows: Fix `NoMethodError` of --daemon under Windows
+  https://github.com/fluent/fluentd/pull/4796
+* windows: Add workaround for unexpected exception
```

Comment:
> windows: Add workaround for unexpected exception

We should not handle this as a bug fix since this issue does not affect Fluentd's actual behavior.